### PR TITLE
Feature fix release notes

### PR DESCRIPTION
**Pull request by jprestop**
When I was adding to the [Release Acceptance Testing discussion](https://github.com/dtcenter/METplus/discussions/2342) using the Release Notes, I found a few problems:

- A duplication of “Bugfix”
- Broken links
- A PR number listed instead of an issue number listed

## Expected Differences ##

- [x] Do these changes introduce new tools, command line arguments, or configuration file options? **[No]**</br>
If **yes**, please describe:</br>

- [x] Do these changes modify the structure of existing or add new output data types (e.g. statistic line types or NetCDF variables)? **[No]**</br>
If **yes**, please describe:</br>

## Pull Request Testing ##

- [x] Describe testing already performed for these changes:</br>
Reviewed the updated release notes [here](https://met.readthedocs.io/en/feature_fix_release_notes/Users_Guide/release-notes.html#met-version-12-0-0-beta5-release-notes-20240710)

- [x] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>
Review the changes

- [x] Do these changes include sufficient documentation updates, ensuring that no errors or warnings exist in the build of the documentation? **[Yes]**

- [x] Do these changes include sufficient testing updates? **[Yes]**

- [x] Will this PR result in changes to the MET test suite? **[No]**</br>
If **yes**, describe the new output and/or changes to the existing output:</br>

- [x] Will this PR result in changes to existing METplus Use Cases? **[No]**</br>
If **yes**, create a new **Update Truth** [METplus issue](https://github.com/dtcenter/METplus/issues/new/choose) to describe them.

- [x] Do these changes introduce new SonarQube findings? **[No]**</br>
If **yes**, please describe:

- [x] Please complete this pull request review by **[20240712]**.</br>

## Pull Request Checklist ##
See the [METplus Workflow](https://metplus.readthedocs.io/en/latest/Contributors_Guide/github_workflow.html) for details.
- [N/A] Review the source issue metadata (required labels, projects, and milestone).
- [x] Complete the PR definition above.
- [x] Ensure the PR title matches the feature or bugfix branch name.
- [x] Define the PR metadata, as permissions allow.
Select: **Reviewer(s)** and **Development** issue
Select: **Milestone** as the version that will include these changes
Select: **Coordinated METplus-X.Y Support** project for bugfix releases or **MET-X.Y.Z Development** project for official releases
- [ ] After submitting the PR, select the :gear: icon in the **Development** section of the right hand sidebar. Search for the issue that this PR will close and select it, if it is not already selected.
- [ ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [ ] Close the linked issue and delete your feature or bugfix branch from GitHub.

### CODE REVIEW

File: docs/Users_Guide/release-notes.rst [hunk 1]
```diff
@@ -22,10 +22,10 @@ MET Version 12.0.0-beta5 Release Notes (20240710)
...
      * Bugfix: Fix the Grid-Stat configuration file to support the MET_SEEPS_GRID_CLIMO_NAME option (`#2601 <https://github.com/dtcenter/MET/issues/2601>`_).
-     * **Bugfix: Fix TC-RMW to correct the tangential and radial wind computations** (`#2841 <https://github.com/dtcenter/MET/issues/#2841>`_).
+     * **Bugfix: Fix TC-RMW to correct the tangential and radial wind computations** (`#2841 <https://github.com/dtcenter/MET/issues/2841>`_).
      * Bugfix: Fix Ensemble-Stat's handling of climo data when verifying ensemble-derived probabilities (`#2856 <https://github.com/dtcenter/MET/issues/2856>`_).
      * **Bugfix: Fix Point2Grid's handling of the -qc option for ADP input files** (`#2867 <https://github.com/dtcenter/MET/issues/2867>`_).
-     * Bugfix: Bugfix: Fix Stat-Analysis errors for jobs using the -dump_row option and the -line_type option with VCNT, RPS, DMAP, or SSIDX (`#2888 <https://github.com/dtcenter/MET/issues/2888>`_).
+     * Bugfix: Fix Stat-Analysis errors for jobs using the -dump_row option and the -line_type option with VCNT, RPS, DMAP, or SSIDX (`2888 <https://github.com/dtcenter/MET/issues/2888>`_).
```
Comment (JohnHalleyGotway) [inline docs/Users_Guide/release-notes.rst:28]:
```suggestion
     * Bugfix: Fix Stat-Analysis errors for jobs using the -dump_row option and the -line_type option with VCNT, RPS, DMAP, or SSIDX (`#2888 <https://github.com/dtcenter/MET/issues/2888>`_).
```

**Review by JohnHalleyGotway — APPROVED** [review]
Julie, apologies for not doing a more thorough review the first time. This time I reviewed the docs at this link:
https://met--2934.org.readthedocs.build/en/2934/Users_Guide/release-notes.html

And I did find 2 additional changes needed, which I added in [this commit](https://github.com/dtcenter/MET/commit/eff8bd9eb7e21b737f4cfd3836890c1016ce137a). We were missing one `#` character and needed to change the PR `#2873` to the issue `#2795`.

Only other question is about issue #2857. It's still open. Ideally it'd be closed. Do you know if that should be closed or if Howard's still working on it? I suppose as long as its actually closed by the official release, that's acceptable.

I approve.

**Comment by hsoh-u** [comment]
Created two new issues (https://github.com/dtcenter/MET/issues/2935 and https://github.com/dtcenter/MET/issues/2936) and closed https://github.com/dtcenter/MET/issues/2857

**Comment by jprestop** [comment]
Thanks @JohnHalleyGotway and @hsoh-u.

**Pull request merged**